headless/package.json

```diff
@@ -1,59 +1,25 @@
 {
 	"name": "headless",
-	"version": "0.0.1",
-	"main": "index.js",
+	"version": "0.1.0",
 	"repository": "https://github.com/ckeditor/ckeditor5-demos",
 	"author": "CKSource (http://cksource.com/)",
 	"license": "GPL-2.0-or-later",
 	"private": true,
-	"devDependencies": {
-		"@babel/core": "^7.20.7",
-		"@babel/preset-env": "^7.20.2",
-		"@babel/preset-react": "^7.18.6",
-		"@ckeditor/ckeditor5-alignment": "40.2.0",
-		"@ckeditor/ckeditor5-autoformat": "40.2.0",
-		"@ckeditor/ckeditor5-basic-styles": "40.2.0",
-		"@ckeditor/ckeditor5-block-quote": "40.2.0",
-		"@ckeditor/ckeditor5-code-block": "40.2.0",
-		"@ckeditor/ckeditor5-core": "40.2.0",
-		"@ckeditor/ckeditor5-dev-utils": "^39.5.0",
-		"@ckeditor/ckeditor5-editor-decoupled": "40.2.0",
-		"@ckeditor/ckeditor5-engine": "40.2.0",
-		"@ckeditor/ckeditor5-enter": "40.2.0",
-		"@ckeditor/ckeditor5-essentials": "40.2.0",
-		"@ckeditor/ckeditor5-heading": "40.2.0",
-		"@ckeditor/ckeditor5-image": "40.2.0",
-		"@ckeditor/ckeditor5-indent": "40.2.0",
-		"@ckeditor/ckeditor5-link": "40.2.0",
-		"@ckeditor/ckeditor5-list": "40.2.0",
-		"@ckeditor/ckeditor5-paragraph": "40.2.0",
-		"@ckeditor/ckeditor5-paste-from-office": "40.2.0",
-		"@ckeditor/ckeditor5-react": "5.0.5",
-		"@ckeditor/ckeditor5-remove-format": "40.2.0",
-		"@ckeditor/ckeditor5-table": "40.2.0",
-		"@ckeditor/ckeditor5-theme-lark": "40.2.0",
-		"@ckeditor/ckeditor5-typing": "40.2.0",
-		"@ckeditor/ckeditor5-ui": "40.2.0",
-		"@ckeditor/ckeditor5-undo": "40.2.0",
-		"@ckeditor/ckeditor5-utils": "40.2.0",
-		"babel-loader": "^9.1.0",
-		"css-loader": "^6.7.3",
-		"postcss": "^8.4.19",
-		"postcss-loader": "^4.3.0",
-		"raw-loader": "^4.0.2",
-		"react": "^18.2.0",
-		"react-dom": "^18.2.0",
-		"sass": "^1.57.1",
-		"sass-loader": "^13.2.0",
-		"style-loader": "^3.3.1",
-		"terser-webpack-plugin": "^4.2.3",
-		"webpack": "^5.75.0",
-		"webpack-cli": "^4.10.0",
-		"webpack-dev-server": "^4.11.1"
+	"type": "module",
+	"engines": {
+		"node": ">=18.0.0"
 	},
 	"scripts": {
-		"build": "webpack --mode production",
-		"build-dev": "webpack --mode development",
-		"start": "webpack serve --open /index.html --mode development"
+		"dev": "vite --open",
+		"build": "vite build",
+		"preview": "vite preview"
+	},
+	"devDependencies": {
+		"@ckeditor/ckeditor5-react": "alpha",
+		"ckeditor5": "alpha",
+		"ckeditor5-premium-features": "alpha",
```

Comment:
We can remove `ckeditor5-premium-features` dependency cause it's not used in this demo.

Reply:
Addressed in 8a36a76.

Reply:
Good catch, thanks for the patch 👍